app/team.go

```diff
@@ -1017,3 +1018,21 @@ func (a *App) SetTeamIconFromFile(teamId string, file multipart.File) *model.App
 
 	return nil
 }
+
+func (a *App) RemoveTeamIcon(teamId string) *model.AppError {
+	team, err := a.GetTeam(teamId)
+	if err != nil {
+		return model.NewAppError("RemoveTeamIcon", "api.team.remove_team_icon.get_team.app_error", nil, err.Error(), http.StatusBadRequest)
+	}
+
+	if result := <-a.Srv.Store.Team().UpdateLastTeamIconUpdate(teamId, 0); result.Err != nil {
+		return model.NewAppError("SetTeamIcon", "api.team.team_icon.update.app_error", nil, result.Err.Error(), http.StatusBadRequest)
+	}
+
+	// manually set time to avoid possible cluster inconsistencies
```

Comment:
For my own education, what does this comment imply?

Reply:
I removed the comment since it's a bit inaccurate (just copied from SetTeamIcon).
Instead of getting the updated team, this is for quick optimistic update of team to be sent via websocket.